# Generated by Django 3.0.2 on 2020-03-12 15:00

import customUser.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('customUser', '0001_initial'),
    ]

    operations = [
        migrations.AlterField(
            model_name='siteuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Erforderlich! Muss der volle Namen sein.', max_length=150, unique=True, validators=[customUser.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
